src/Speech/BrowserSpeech.ts

```diff
@@ -0,0 +1,266 @@
+import { ISpeechRecognizer, ISpeechSynthesizer, Action, Func } from "./SpeechInterface"
```

Comment:
nit-pick: please use single quotes around imports rather than double quotes (for style consistency).